src/Data/NormalKey.purs

```diff
@@ -101,6 +97,8 @@ instance ordNormalKey :: Ord NormalKey where
   compare (NormalKey "Shift") _ = LT
   compare _ (NormalKey "Shift") = GT
   compare (NormalKey "Enter") (NormalKey "Enter") = EQ
+  compare _ (NormalKey "Enter") = GT
+  compare (NormalKey "Enter") _ = LT
   compare (NormalKey x) (NormalKey y) = compare x y
```

Comment:
It's a little bit hard to verify that this is correct just by looking at it. Another way that you could make it more "obviously" correct is if you factored the implementation of `compare` through an injection into a type that has an obvious `Ord` instance.

Implement an injection/encoding from `NormalKey` into a type that has an obvious `Ord` instance, like the natural numbers or integers. So, if you code up an *injective* function from `NormalKey` into the integers, then you can re-use the `Ord` instance for `Integer`; this way, you can be sure that you have followed the laws.